src/Spryker/Zed/UserMerchantPortalGui/Presentation/Components/app/change-field-overlay/change-field-overlay.component.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { TestBed } from '@angular/core/testing';
+import { createComponentWrapper, getTestingForComponent } from '@mp/zed-ui/testing';
+import { ChangeFieldOverlayComponent } from './change-field-overlay.component';
```

Comment:
The appropriate environment should be set up to check it or change the test according to the environment.

Reply:
Copied the test file from password component